compiler/src/dotty/tools/dotc/transform/init/Objects.scala

```diff
@@ -68,11 +69,21 @@ import dotty.tools.dotc.core.Flags.AbstractOrTrait
  *
  */
 class Objects(using Context @constructorOnly):
-  val immutableHashSetBuider: Symbol = requiredClass("scala.collection.immutable.HashSetBuilder")
+  val immutableHashSetNode: Symbol = requiredClass("scala.collection.immutable.SetNode")
   // TODO: this should really be an annotation on the rhs of the field initializer rather than the field itself.
-  val HashSetBuilder_rootNode: Symbol = immutableHashSetBuider.requiredValue("rootNode")
-
-  val whiteList = Set(HashSetBuilder_rootNode)
+  val SetNode_EmptySetNode: Symbol = Denotations.staticRef("scala.collection.immutable.SetNode.EmptySetNode".toTermName).symbol
+  val immutableHashSet: Symbol = requiredModule("scala.collection.immutable.HashSet")
+  val HashSet_EmptySet: Symbol = Denotations.staticRef("scala.collection.immutable.HashSet.EmptySet".toTermName).symbol
+  val immutableVector: Symbol = requiredModule("scala.collection.immutable.Vector")
+  val Vector_EmptyIterator: Symbol = immutableVector.requiredValue("emptyIterator")
+  val immutableMapNode: Symbol = requiredModule("scala.collection.immutable.MapNode")
+  val MapNode_EmptyMapNode: Symbol = immutableMapNode.requiredValue("EmptyMapNode")
+  val immutableHashMap: Symbol = requiredModule("scala.collection.immutable.HashMap")
+  val HashMap_EmptyMap: Symbol = immutableHashMap.requiredValue("EmptyMap")
+  val immutableLazyList: Symbol = requiredModule("scala.collection.immutable.LazyList")
+  val LazyList_empty: Symbol = immutableLazyList.requiredValue("_empty")
+
+  val whiteList: Set[Symbol] = Set()
```

Comment:
Did you mean it to be empty?

Reply:
Sorry, I forgot to push this. It has been corrected.